documentation/docs/.vuepress/config.ts.fc0d442e.mjs

```diff
@@ -0,0 +1,61 @@
aWcoe1xuICBiYXNlOiBcIi9nZHgtYWdyZWVtZW50cy10cmFja2VyL1wiLFxuICBsYW5nOiBcImVuLVVTXCIsXG4gIHRpdGxlOiBcIkdEWCBBZ3JlZW1lbnRzIFRyYWNrZXJcIixcbiAgZGVzY3JpcHRpb246IFwiRGV2ZWxvcGVyIERvY3VtZW50YXRpb24gZm9yIEdEWCBBZ3JlZW1lbnRzIFRyYWNrZXJcIixcbiAgdGhlbWU6IGRlZmF1bHRUaGVtZSh7XG4gICAgbG9nbzogXCIvaW1hZ2VzL0JDSURfSF9yZ2JfcG9zLnBuZ1wiLFxuICAgIC8vIHNpZGViYXIgYXJyYXlcbiAgICAvLyBhbGwgcGFnZXMgd2lsbCB1c2UgdGhlIHNhbWUgc2lkZWJhclxuICAgIHNpZGViYXI6IFtcbiAgICAgIFwiL2d1aWRlL1JFQURNRS5tZFwiLFxuICAgICAgLy8gU2lkZWJhckl0ZW1cbiAgICAgIHtcbiAgICAgICAgdGV4dDogXCJHZXR0aW5nIFN0YXJ0ZWRcIixcbiAgICAgICAgY29sbGFwc2libGU6IHRydWUsXG4gICAgICAgIGNoaWxkcmVuOiBbXG4gICAgICAgICAgXCIvZ3VpZGUvR2V0dGluZ1N0YXJ0ZWQvZGVwbG95X2xvY2FsbHlcIlxuICAgICAgICBdXG4gICAgICB9LCBcbiAgICAgIHtcbiAgICAgICAgdGV4dDogXCJGcm9udGVuZFwiLFxuICAgICAgICBjb2xsYXBzaWJsZTogdHJ1ZSxcbiAgICAgICAgY2hpbGRyZW46IFtcbiAgICAgICAgICBcIi9ndWlkZS9Gcm9udGVuZC9yZWFjdF9jb21wb25lbnRzL0FjY29yZGlvblwiLFxuICAgICAgICAgIFwiL2d1aWRlL0Zyb250ZW5kL3JlYWN0X2NvbXBvbmVudHMvQnVkZ2V0RGlzcGxheVwiLFxuICAgICAgICAgIFwiL2d1aWRlL0Zyb250ZW5kL3JlYWN0X2NvbXBvbmVudHMvQnVkZ2V0RGlzcGxheVwiLFxuICAgICAgICAgIFwiL2d1aWRlL0Zyb250ZW5kL3JlYWN0X2NvbXBvbmVudHMvQ2hpcE5hdlwiLFxuICAgICAgICAgIFwiL2d1aWRlL0Zyb250ZW5kL3JlYWN0X2NvbXBvbmVudHMvQ29uZmlybWF0aW9uRGlhbG9nXCIsXG4gICAgICAgICAgXCIvZ3VpZGUvRnJvbnRlbmQvcmVhY3RfY29tcG9uZW50cy9EQkxvY2tcIixcbiAgICAgICAgICBcIi9ndWlkZS9Gcm9udGVuZC9yZWFjdF9jb21wb25lbnRzL0RlYnVnXCIsXG4gICAgICAgIF0sXG4gICAgICB9LFxuICAgICAge1xuICAgICAgICB0ZXh0OiBcIkJhY2tlbmRcIixcbiAgICAgICAgY29sbGFwc2libGU6IHRydWUsXG4gICAgICAgIGNoaWxkcmVuOiBbXCIvZ3VpZGUvQmFja2VuZC9iYWNrZW5kX2FwaS5tZFwiXSxcbiAgICAgIH0sXG4gICAgICB7XG4gICAgICAgIHRleHQ6IFwiT3BlbnNoaWZ0XCIsXG4gICAgICAgIGNvbGxhcHNpYmxlOiB0cnVlLFxuICAgICAgICBjaGlsZHJlbjogW1wiL2d1aWRlL29wZW5zaGlmdC9jaGFuZ2VfcGFzc3dvcmRcIl0sXG4gICAgICB9LFxuICAgICAgLy8gc3RyaW5nIC0gcGFnZSBmaWxlIHBhdGhcbiAgICBdLFxuICB9KSxcbiAgcGx1Z2luczogW1xuICAgIHNlYXJjaFBsdWdpbih7XG4gICAgICAvLyBvcHRpb25zXG4gICAgfSksXG4gICAgXG4gICAgY29weUNvZGVcbiAgXSxcbn0pO1xuIl0sCiAgIm1hcHBpbmdzIjogIjtBQUFvYSxTQUFTLG9CQUFvQjtBQUVqYyxTQUFTLHdCQUF3QjtBQUNqQyxTQUFTLG9CQUFvQjtBQUM3QixPQUFRLGNBQWU7QUFFdkIsSUFBTyxpQkFBUSxpQkFBaUI7QUFBQSxFQUM5QixNQUFNO0FBQUEsRUFDTixNQUFNO0FBQUEsRUFDTixPQUFPO0FBQUEsRUFDUCxhQUFhO0FBQUEsRUFDYixPQUFPLGFBQWE7QUFBQSxJQUNsQixNQUFNO0FBQUE7QUFBQTtBQUFBLElBR04sU0FBUztBQUFBLE1BQ1A7QUFBQTtBQUFBLE1BRUE7QUFBQSxRQUNFLE1BQU07QUFBQSxRQUNOLGFBQWE7QUFBQSxRQUNiLFVBQVU7QUFBQSxVQUNSO0FBQUEsUUFDRjtBQUFBLE1BQ0Y7QUFBQSxNQUNBO0FBQUEsUUFDRSxNQUFNO0FBQUEsUUFDTixhQUFhO0FBQUEsUUFDYixVQUFVO0FBQUEsVUFDUjtBQUFBLFVBQ0E7QUFBQSxVQUNBO0FBQUEsVUFDQTtBQUFBLFVBQ0E7QUFBQSxVQUNBO0FBQUEsVUFDQTtBQUFBLFFBQ0Y7QUFBQSxNQUNGO0FBQUEsTUFDQTtBQUFBLFFBQ0UsTUFBTTtBQUFBLFFBQ04sYUFBYTtBQUFBLFFBQ2IsVUFBVSxDQUFDLCtCQUErQjtBQUFBLE1BQzVDO0FBQUEsTUFDQTtBQUFBLFFBQ0UsTUFBTTtBQUFBLFFBQ04sYUFBYTtBQUFBLFFBQ2IsVUFBVSxDQUFDLGtDQUFrQztBQUFBLE1BQy9DO0FBQUE7QUFBQSxJQUVGO0FBQUEsRUFDRixDQUFDO0FBQUEsRUFDRCxTQUFTO0FBQUEsSUFDUCxhQUFhO0FBQUE7QUFBQSxJQUViLENBQUM7QUFBQSxJQUVEO0FBQUEsRUFDRjtBQUNGLENBQUM7IiwKICAibmFtZXMiOiBbXQp9Cg==
```

Comment:
Is this needed still? The commented out source map url

Reply:
i believe this was done automatically, i had nothing to do with this file